study.md

```diff
@@ -31,7 +31,7 @@ manipulates it by sorting it and then reversing it in place (i.e., modifying the
 reference to the original).
 
 ```ruby
-# your answer here
+[23, 56, 3, 7].sort.reverse
```

Comment:
Since we want to the modify the array in place, the invocations of `sort` and `reverse` should be `sort!` and `reverse!`, respectively.